src/metadata/v55.json

```diff
@@ -1,4 +1,11 @@
 [
+  {
+    "directoryName": "customindex",
```

Comment:
**remark:** I think it should be added in the list of Custom Object children line [156](https://github.com/scolladon/sfdx-git-delta/pull/324/files#diff-f6449921ca6fbe6bc2cd8d44b67a557af1fc3778f5c7ee415ddc516e105014ceR156) as well

Reply:
Actually, customindex folder is not nested into the parent object folder. The structure is like:

├── default
    ├── customindex
    │    └── Account.Fax.indx-meta.xml
    └── objects
        └── Account
            └── fields
                └──  Fax.field-meta.xml

Based on this, I would believe it should not be added, but maybe I am misunderstanding something.

Reply:
Ok clear, thanks for the update